web/packages/teleterm/src/ui/TopBar/Clusters/ClusterSelector/ClusterSelector.tsx

```diff
@@ -78,15 +78,6 @@ const Container = styled.button`
   &:hover,
   &:focus {
     opacity: 1;
-    border-color: ${props => props.theme.colors.light};
+    background: ${props => props.theme.colors.spotBackground[0]};
```

Comment:
Another candidate to get replaced with `levels.popout`.

Reply:
Since this is to highlight a focused state, we should use a `spotBackground`